src/components/editor.tsx

```
@@ -622,18 +622,18 @@ const Editor = ({
+					<div className="p-1.5 md:p-2">
+						<div className="group/image relative flex size-[50px] md:size-[62px] items-center justify-center">
 							<Hint label="Remove image">
 								<button
-									className="absolute -right-2.5 -top-2.5 z-[4] hidden size-6 items-center justify-center rounded-full border-2 border-white bg-black/70 text-white hover:bg-black group-hover/image:flex"
+									className="absolute -right-2 -top-2 md:-right-2.5 md:-top-2.5 z-[4] hidden size-5 md:size-6 items-center justify-center rounded-full border-2 border-white bg-black/70 text-white hover:bg-black group-hover/image:flex"
 									onClick={() => {
 										setImage(null);
 
 										imageElementRef.current!.value = "";
 									}}
 								>
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)